### ISSUE TITLE
Movie Recommendation

### ISSUE THREAD

**Issue by sagedemage** [sagedemage/movie_recommendation_system#21]
**Describe the feature**
Change the way how the main program recommends a movie. Adjust the MovieRecommendation model class where it should only apply the linear transformation (nn.Linear) two times and apply the rectified linear unit (nn.ReLU) once.

## Advice
It is probably a good idea to use cast the model output, logits, as nn.ReLU (or nn.Softplus) to ensure the output of model is always positive.

Code
```
...
logits = saved_model(data_movie_ids)
...
print("logits:")
print(logits)
print("nn.ReLU:")
print(nn.ReLU()(logits))
```

Output
```
logits:
tensor([-7.0684,  8.5576, 26.8145, 25.0905], device='cuda:0',
       grad_fn=<ViewBackward0>)
nn.ReLU:
tensor([ 0.0000,  8.5576, 26.8145, 25.0905], device='cuda:0',
       grad_fn=<ReluBackward0>)
```

## Resources
- https://pytorch.org/docs/stable/generated/torch.nn.ReLU.html#torch.nn.ReLU
- https://pytorch.org/docs/stable/generated/torch.nn.Softplus.html#torch.nn.Softplus
- https://pytorch.org/docs/stable/nn.html#non-linear-activations-weighted-sum-nonlinearity
- https://datascience.stackexchange.com/questions/41046/how-to-force-pytorch-model-to-predict-only-positive-values
- https://pytorch.org/tutorials/beginner/basics/buildmodel_tutorial.html
- https://pytorch.org/tutorials/beginner/basics/optimization_tutorial.html
- https://pytorch.org/docs/stable/generated/torch.argmax.html#torch.argmax
- https://pytorch.org/docs/stable/generated/torch.nn.Sequential.html#torch.nn.Sequential
- https://pytorch.org/docs/stable/generated/torch.nn.Linear.html#torch.nn.Linear
